projects/elife.yaml

```diff
@@ -404,6 +404,9 @@ journal:
                 engine: redis
                 clusters: 2
                 overrides:
+                    1:
+                        configuration:
+                            maxmemory-policy: volatile-lru
```

Comment:
We should make this the default and instead override those that have the old default.

Reply:
Where is everywhere for now?

Reply:
It's easy to override everywhere, as more groups will be created; then the problem is changing the default, which is still immutable. I think I have to first add a removal of the default parameter group in case it is overridden everywhere.